### PR TITLE
Add NCV test cases for Glidepath

### PR DESCRIPTION
**Pull request by qswawrq**

### CODE REVIEW

**Comment by qswawrq** [comment]
Total Plans: 2268
Failed Plans: 0
Percent Failed: 0.0
Average Planning Time ms: 148.4775132275132

File: carmajava/signal_plugin/src/test/java/gov/dot/fhwa/saxton/carma/signal_plugin/MockCollisionChecker.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    }
+    
+    public void setPredictedTrajectory(Node startNode, Node endNode) {
+        ncvStartNode = startNode;
+        ncvEndNode = endNode;
+    }
+    
+    
+    private boolean onSegment(Node p, Node q, Node r) {
+        return q.getTime()     >= Math.max(p.getTime(), r.getTime())     &&
```
Comment (msmcconnell) [inline carmajava/signal_plugin/src/test/java/gov/dot/fhwa/saxton/carma/signal_plugin/MockCollisionChecker.java]: I think you have the signs of this check flipped
https://www.geeksforgeeks.org/check-if-two-given-line-segments-intersect/

Comment (qswawrq) [reply carmajava/signal_plugin/src/test/java/gov/dot/fhwa/saxton/carma/signal_plugin/MockCollisionChecker.java]: You are right...

**Comment by qswawrq** [comment]
Fixed.

**Pull request merged**